code/modules/mob/living/simple_animal/hostile/retaliate/cavern.dm

```diff
@@ -47,9 +50,17 @@
 	see_in_dark = 8
 	see_invisible = SEE_INVISIBLE_NOLIGHTING
 
+	butchering_products = list(/obj/item/grenade/tesla/bio = 1)
```

Comment:
I think getting a grenade for every mob butchered is a bit too much considering how many dwellers there may be. Also how tf do you get an entire grenade from a dweller?